src/xmipp/applications/programs/mpi_write_test/mpi_write_test.cpp

```diff
@@ -43,6 +43,7 @@
 #include "parallel/xmipp_mpi.h"
 #include <core/args.h>
 #include <core/xmipp_program.h>
+#include "core/xmipp_image_generic.h"
```

Comment:
Although they work, is it not better to do includes from different "libraries" with angles?

Reply:
with `""`, preprocessor should start searching in the current location, unlike with `<>`, which should search 'standard' locations (like `/usr/local/include`) first, i.e. `""` is recommended for proprietary headers, while `<>` is recommended for standard library headers